packages/forklift-console-plugin/src/modules/Providers/views/migrate/ProvidersCreateVmMigration.style.css

```diff
@@ -21,8 +43,17 @@
 
 .forklift--create-vm-migration-plan--section-header {
   margin-bottom: 0;
+  justify-content: start;
```

Comment:
So that the `NM` and `SM` labels in the headers appear next to the text and are vertically centered